src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -283,6 +283,11 @@ export class ResultsViewPageStore {
 
     public mutationAnnotationSettings:MutationAnnotationSettings;
 
+    @observable selectedEnrichmentMutationProfile: MolecularProfile;
```

Comment:
use @observable.ref here.  @observable makes the object deeply @observable, which is unnecessary since the data is/should be immutable.